src/home/form/LoginForm.jsx

```diff
@@ -26,28 +28,35 @@ class Loginpage extends React.Component {
         this.setState({password: e.target.value});
     }
 
+    setErrorMessage() {
+        this.setState({errorMessage:'error-message'});
+    }
+
     sendToServer(e) {
         console.log('LoginForm is submitting the DATA MAN');
         e.preventDefault();
-
+        var self = this;
```

Comment:
Lol is this a js thing? Seems unnecessary

Reply:
yeah haha it's an annoying javascript thing where I need to set 'this' to a new variable or else it isn't in the scope